src/main/java/step3/controller/CarRacingController.java

```diff
@@ -0,0 +1,48 @@
+package step3.controller;
+
+import step3.model.Car;
+import step3.view.InputView;
+import step3.view.ResultView;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+public class CarRacingController {
+    Random random = new Random();
+    InputView inputView = new InputView();
+    ResultView resultView = new ResultView();
+
+    private final int MAX_BOUND = 10;
+    private final int MOVE_STANDARD = 4;
+
+    // 난수 생성
+    public int randomNumberGenerator(){
+        return random.nextInt(MAX_BOUND);
+    }
+
+    // 차 움직임 판별
+    public void moveCar(int num, Car car) {
+        if (num >= MOVE_STANDARD) {
+            car.move();
+        }
+    }
+
+    // 레이싱 게임
+    public void racing() {
+        int numberOfCars = inputView.askNumberOfCars();
+        int numberOfTrys = inputView.askNumberOfTrys();
```

Comment:
원시값은 포장하는것이 좋습니다. 한번 Try부분도 포장 해볼까요?

Reply:
```suggestion
        int numberOfTrys = new InputView("TRY");
```

제가 제대로 이해한지 모르겠습니다.
이런식으로 생성자에 들어가는 변수에 따라 다르게 주어주면 되는걸까요?

Reply:
객체지향 생활체조엔, 원시값을 포장하라 라는 말이 있습니다. 원시값을 포장하라는 것은 단순히 Try와 같은 시도를 int로 쓰지 말고 객체화를 하라는 의미인데요. Try라는 시도 횟수도 충분히 원시값으로 바라볼수 있지 않을까 했습니다.

Reply:
아아 그 의미셨군요, 이해했습니다 ㅎㅎ